examples/zookeeper/config-and-run.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash
 
+# Make sure conf folder exists
+if [[ ! -d /opt/zookeeper/conf ]]; then
+  mkdir -p /opt/zookeeper/conf
+fi
+
+# Copy config files if conf folder is empty
+cp --no-clobber /opt/zookeeper/conf.template/* /opt/zookeeper/conf/
```

Comment:
this is going to copy any non-conflicting files.  you sure that's what you want?  The previous logic only copied files if the folder was empty. Now you're going to merge content.